sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_registry_client.py

```diff
@@ -15,6 +15,7 @@
 )
 from azure.core.tracing.decorator import distributed_trace
 from azure.core.tracing.decorator_async import distributed_trace_async
+from msrestazure.azure_cloud import AZURE_PUBLIC_CLOUD
```

Comment:
We shouldn't have a dependency on msrestazure?

Reply:
We should just define a constant for this.

Reply:
Have removed the usage of this package in testcase.py, so it's no longer a required dependency in dev.